utils/log-target.js

```diff
@@ -58,49 +77,19 @@ module.exports = class LoggingTarget {
 
   /** Construct the service bucket name for Stackdriver logging. */
   get serviceName() {
-    // TODO: Drastically reduce combinatoral explosion of buckets.
-    const {
-      singlePassType,
-      referrer,
-      runtime,
-      thirdParty,
-      binaryType,
-      canary,
-      assert,
-      expected,
-    } = this.opts;
-    const name = ['default'];
-
-    if (singlePassType) {
-      name.push(singlePassType);
-    }
-    name.push(CDN_REGEX.test(referrer) ? 'cdn' : 'origin');
-
-    if (runtime) {
-      name.push(runtime);
-    } else if (thirdParty) {
-      name.push('3p');
-    } else {
-      name.push('1p');
-    }
+    const { referrer, version, expected } = this.opts;
```

Comment:
just a qq on my end, do we assert or verify the scheme of the version anywhere currently?

Reply:
We validate against https://cdn.ampproject.org/rtv/metadata and drop errors with any version not listed